excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/GenericLayoutParser.java

```diff
@@ -84,6 +91,17 @@ private void addParseLayoutCaseField(final ParseResult<GenericLayoutEntity> resu
         }
     }
 
+    private boolean hasDuplicateUserRoleDefinition(List<DefinitionDataItem> layoutItems) {
+        return layoutItems
+            .stream()
+            .filter(ddi ->
+                layoutItems.stream().filter(item -> (StringUtils.isNotEmpty(ddi.getString(USER_ROLE)) ? ddi.getString(USER_ROLE).equalsIgnoreCase(item.getString(USER_ROLE)) : false)
```

Comment:
rather than filter, could we use anyMatch which returns a boolean?

Reply:
done 👍